### PR TITLE
Update navigation

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-wallet/src/app/getDefaultPathForWalletState.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const getDefaultPathForWalletState = (walletState: any) => {
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/app/getDefaultPathForWalletState.ts]: any?

File: universal-login-wallet/src/ui/react/CreateAccount/CreateAccount.tsx [hunk 2]
```diff
@@ -5,15 +5,17 @@ import {useServices, useWalletConfig} from '../../hooks';
...
-  const {sdk} = useServices();
+export const CreateAccount = () => {
+  const {sdk, walletService} = useServices();
   const walletConfig = useWalletConfig();
+  const history = useHistory();
 
+  const onCreateClick = async (ensName: string) => {
+    await walletService.createFutureWallet(ensName);
+    history.push('/create');
+  };
```
Comment (xitronix) [inline universal-login-wallet/src/ui/react/CreateAccount/CreateAccount.tsx]: Enter

File: universal-login-wallet/src/ui/react/WalletRoute.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        : (<Redirect
+          to={{
+            pathname: getDefaultPathForWalletState(walletState.kind),
+            state: {from: props.location},
+          }}
+        />)}
+  />
+);
+
+const mapping = {
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/WalletRoute.tsx]: urlMapping

Comment (xitronix) [inline universal-login-wallet/src/ui/react/WalletRoute.tsx]: move to constants

File: universal-login-wallet/src/ui/react/ConnectAccount/ConnectSelector.tsx [hunk 4]
```diff
@@ -22,28 +22,35 @@ export const ConnectSelector = ({setName, setConnectModal}: ConnectSelectorProps
...
-            domains={walletConfig.domains}
-            actions={[WalletSuggestionAction.connect]}
-            className="jarvis"
-            placeholder="e.g. satoshi"
-          />
-          <Link to="/welcome" className="button-secondary connect-account-cancel">Cancel</Link>
+
+    <div className="main-bg">
+      <div className="box-wrapper">
+        <div className="box">
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/ConnectAccount/ConnectSelector.tsx]: Maybe wrapper?

File: universal-login-wallet/src/ui/react/ConnectAccount/ConnectSelector.tsx [hunk 5]
```diff
@@ -22,28 +22,35 @@ export const ConnectSelector = ({setName, setConnectModal}: ConnectSelectorProps
...
+          <div>
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/ConnectAccount/ConnectSelector.tsx]: I believe this div is unnecessary

File: universal-login-wallet/src/ui/react/ConnectAccount/ConnectSelector.tsx (hunk 4 above)
Comment (xitronix) [reply universal-login-wallet/src/ui/react/ConnectAccount/ConnectSelector.tsx]: I think i is not for this pr. We should create ticket/PR with refactor stylings

File: universal-login-wallet/src/ui/react/WalletRoute.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+export interface WalletRouteProps extends RouteProps {
+  walletState: WalletState;
+  component: React.ComponentType<RouteComponentProps<any>> | React.ComponentType<any>;
+}
+
+export const WalletRoute = ({walletState, component, render, ...restProps}: WalletRouteProps) => (
+  <Route
+    {...restProps}
+    render={props =>
+      urlMapping[walletState.kind as 'None'].filter(path => props.location.pathname.includes(path)).length > 0
```
Comment (marekkirejczyk) [inline universal-login-wallet/src/ui/react/WalletRoute.tsx]: Extract to app/

File: universal-login-wallet/src/ui/react/Home/HomeScreen.tsx [hunk 7]
```diff
@@ -24,42 +26,42 @@ const HomeScreen = () => {
...
               </Route>
-              <Route path="/backup">
+              <Route path={join(basePath, 'backup')}>
                 <BackupCodes
                   deployedWallet={walletService.getDeployedWallet()}
                   className="jarvis-backup"
                 />
               </Route>
-              <Route path="/topUp">
+              <Route path={join(basePath, 'topUp')}>
```
Comment (marekkirejczyk) [inline universal-login-wallet/src/ui/react/Home/HomeScreen.tsx]: basePath => currentPath
extract `pathFor(currentPath, 'topup')`, that uses `replaceSecondSegment` or similar

**Pull request merged**